content/help/faq/security/mysql-with-mtls.md

```diff
@@ -0,0 +1,60 @@
+---
+title: MySQL Connectivity Troubleshooting
+description: Integrate VMs and bare metal hosts into an Istio mesh deployed on Kubernetes.
+weight: 95
+keywords: [mysql,mtls]
+---
+
+# MySQL Connectivity Troubleshooting
+
+You may find MySQL can't be connected after installing Istio. This is because that `PERMISSIVE` mode,
+default option `istio-demo.yaml` install, does not work with MySQL.
+
+We have two options to solve the problem.
+
+## Disable Mutual TLS
+
+Choose this option if you don't want Istio mutual TLS. You achieve this by disable mutual TLS on MySQL
+service explicitly.
+
+```bash
```

Comment:
Please use a {{< text >}} block here. See https://istio.io/create for details.

Reply:
Done. Just curious what's the benefits of using {{<text>}}? Is that a syntax we create ourselves?

Reply:
{{< text >}} is custom syntax we use. If does error checking for a few cases, plus supports options like downloadas outputis, snippet, and expandlinks